crates/compiler/builtins/roc/Num.roc

```diff
@@ -1291,7 +1291,7 @@ toF64Checked : Num * -> Result F64 [OutOfBounds]*
 ## >>> Num.isNaN (Num.pow -1 0.5)
 ##
 ## *NaN* is unusual from other numberic values in that:
-## * *NaN* is not equal to any other number, even itself. [Bool.isEq] always returns `Bool.false` if either argument is *NaN*.
+## * *NaN* is not equal to any other number, even itself. [Eq.isEq] always returns `Bool.false` if either argument is *NaN*.
```

Comment:
Doesn't `isEq` still come from the `Bool` module?
This change seems to suggest there's an Eq module but I don't see one.

Reply:
Oops, good catch Brian